photon-lib/src/test/scala/com/linkedin/photon/ml/util/VectorUtilsTest.scala

```diff
@@ -105,7 +105,7 @@ class VectorUtilsTest {
 
   @Test(dataProvider = "featureValsProvider")
   def testSparseBreezeToMllib(featureVals: Array[Double]): Unit = {
-    val dim = featureVals.length
+    val dim = featureVals.length + 1
```

Comment:
@joshvfleming Adding one to the dimension of `SparseVector` during conversion testing to make sure we're "missing" the final index and thus have fewer indices/values than the size.